docker/docker-compose.yml

```diff
@@ -8,6 +8,8 @@ services:
     volumes:
       - ${UPLOAD_LOCATION}:/usr/src/app/upload
       - /etc/localtime:/etc/localtime:ro
+    # Any external library import paths go here, for example:
```

Comment:
I think we have documentation to indicate this so we don't need to add these lines in the docker-compose file to keep them minimal

Reply:
done!